_tutorials/2016-05-16-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,77 @@
+---
+published: true
+date: "2016-05-16 09:08 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64"
+author: "Lisa Roach, Ric Wellum"
```

Comment:
One author per tutorial, I'm afraid.  Lisa, since you've been added to the database, use "Lisa Roach" in the Author field